### ISSUE TITLE
generated texts are the same as input

### ISSUE THREAD

**Issue by shenmeguimlf** [minimaxir/gpt-2-simple#184]
[quotes.txt](https://github.com/minimaxir/gpt-2-simple/files/4346403/quotes.txt)
[gpt2_gentext_20200317_125607.txt](https://github.com/minimaxir/gpt-2-simple/files/4346404/gpt2_gentext_20200317_125607.txt)
quotes is my input text dataset, the results generated(using the 124M model) are just repeating the quotes mostly the same as original, or slightly modified version, what's wrong with it?

**Comment by miheico** [comment]
same problem.

**Comment by GeoMmm** [comment]
Yep, same here




Get Outlook for Android







On Sat, Mar 21, 2020 at 3:11 PM +0100, "miheico" <notifications@github.com> wrote:















same problem.

**Comment by fillerInk** [comment]
I got the same problem. I thought it was because of my dataset, which has only 1130 lines

**Comment by SeanBannister** [comment]
Having the same problem and I'm guessing it's due to a small data set used for fine-tuning. In my case I was using 20 pieces of data (each between a <|startoftext|> and <|endoftext|>) for a total of 62KB of data.

When I fed in 100,000 pieces of data I found it generating uniqe content.

**Comment by AndreyStille** [comment]
> Having the same problem and I'm guessing it's due to a small data set used for fine-tuning. In my case I was using 20 pieces of data (each between a <|startoftext|> and <|endoftext|>) for a total of 62KB of data.
> 
> When I fed in 100,000 pieces of data I found it generating uniqe content.

Hello
Have you solved the problem?

**Comment by lognorman20** [comment]
> > Having the same problem and I'm guessing it's due to a small data set used for fine-tuning. In my case I was using 20 pieces of data (each between a <|startoftext|> and <|endoftext|>) for a total of 62KB of data.
> > When I fed in 100,000 pieces of data I found it generating uniqe content.
> 
> Hello
> Have you solved the problem?

I solved this issue by increasing my dataset size. You could also try decreasing the learning rate of your model
